src/View/Helper/TimeHelper.php

```diff
@@ -33,6 +34,17 @@ class TimeHelper extends Helper
     use StringTemplateTrait;
 
     /**
+     * Default config for this class
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'defaultLocale' => null,
+        'defaultFormat' => null,
+        'defaultOutputTimezone' => null,
+    ];
```

Comment:
It does not make sense to me to have 2 places for configuring the same. The I18n class already has this info